api/v1/authentication_types.go

```diff
@@ -81,6 +81,12 @@ type AuthenticationOIDC struct {
 	// Deprecated. Please use Authentication.Spec.GroupsPrefix instead.
 	// +optional
 	GroupsPrefix string `json:"groupsPrefix,omitempty"`
+
+	// Some providers do not include the claim "email_verified" when there is no verification in the user enrollment
+	// process or if they are acting as a proxy for another IdP. By default those tokens are deemed invalid.
+	// To skip this check, set the value to true.
+	// +optional
+	InsecureSkipEmailVerified *bool `json:"insecureSkipEmailVerified,omitempty"`
```

Comment:
I don't think this should be a bool to match K8s API conventions.
Maybe `Email: [Verify|SkipInsecure]`... would that make sense?

Reply:
I feel like this is something that should be defaulted and written back to the datastore also.

Reply:
Shouldn't this be `Ensure` rather than `Insure`? 

Actually, couldn't it just be `SkipEmailVerified`?

Are there other similar fields we might introduce in the future? should we have a sub-structure of some sort to group these?

Reply:
Ignore my previous comment, I am still waking up.